### PR TITLE
Change default values to have predictable default state

### LINKED ISSUE #4713: [BUG] Clicking next to article makes it collapse
Issue:
When clicking next to an open article (and also when selecting text and releasing the mouse button in this area), the article collapses. I'm reporting this as a bug since there is no visual feedback ("click"-cursor) for this, unlike on the article header.

1. Open article
2. Click on the margin of the content < div> (see screenshot)
3. Article collapses as if its header was clicked

Expected behavior:
Nothing should happen.

image

Environment information (please complete the following information):
- Device: PC with a large enough screen
- OS: any (tested on Windows 10 and EndeavourOS)
- Browser: any (tested on Chrome 106 and Firefox 105)
- FreshRSS version: 1.20.0
- PHP version: 7.4.30

Suggested solution:
Make only the < li> "item title" clickable for the collapsing behaviour and not the whole < div> "flux active current" as I guess it is the case now.

PS: thanks for your great work, FreshRSS is an awesome piece of software!

### PR DESCRIPTION
**Pull request by aledeg**
Closes #4713

Changes proposed in this pull request:

- Change default values to have predictable default state

How to test the feature manually:

1. Create a new user and check its configuration

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

**Review by Alkarex — APPROVED** [review]

File: config-user.default.php [hunk 1]
```diff
@@ -41,7 +41,7 @@
 	'hide_read_feeds' => true,
 	'onread_jump_next' => true,
 	'lazyload' => true,
-	'sides_close_article' => true,
+	'sides_close_article' => false,
```
Comment (math-GH) [inline config-user.default.php:44]: "Clicking outside of article text area closes the article"

File: config-user.default.php [hunk 2]
```diff
@@ -53,13 +53,13 @@
 	'sort_order' => 'DESC',
 	'anon_access' => false,
 	'mark_when' => array (
-		'article' => true,
+		'article' => false,
```
Comment (math-GH) [inline config-user.default.php]: "when the article is viewed"

File: config-user.default.php [hunk 3]
```diff
@@ -53,13 +53,13 @@
...
 		'gone' => false,
 		'max_n_unread' => false,
 		'reception' => false,
 		'same_title_in_feed' => false,
-		'scroll' => true,
-		'site' => true,
+		'scroll' => false,
+		'site' => false,
```
Comment (math-GH) [inline config-user.default.php]: " while scrolling "
"when the article is opened on its original website "

File: config-user.default.php (hunk 1 above)
Comment (math-GH) [reply config-user.default.php:44]: I like it

File: config-user.default.php (hunk 2 above)
Comment (math-GH) [reply config-user.default.php]: I do not like this change

File: config-user.default.php (hunk 3 above)
Comment (math-GH) [reply config-user.default.php]: first: I like
second: I do not like this change

File: config-user.default.php (hunk 2 above)
Comment (Frenzie) [reply config-user.default.php]: I think true is probably a more expected default for this one, yeah.

File: config-user.default.php (hunk 3 above)
Comment (Frenzie) [reply config-user.default.php]: Having the second as true may be worse than the outside of text area setting. You know you're clicking on it, but you may not even realize you're accidentally marking things as read in the background.

But you could make an argument that it's similar to opening the article within the FreshRSS UI.

File: config-user.default.php (hunk 2 above)
Comment (Alwaysin) [reply config-user.default.php]: I agree

File: config-user.default.php (hunk 3 above)
Comment (Alwaysin) [reply config-user.default.php]: No strong opinion on this one but I join math, I think it makes more sense to mark it as read when you open the article on the original website.

File: config-user.default.php (hunk 2 above)
Comment (aledeg) [reply config-user.default.php]: Fixed

File: config-user.default.php (hunk 3 above)
Comment (aledeg) [reply config-user.default.php]: fixed

**Review by Frenzie — APPROVED** [review]

**Pull request merged**